src/reading.rs

```diff
@@ -122,7 +128,7 @@ impl PageInfo {
 	/// (spans page borders), this returns `false`.
 	fn is_last_pck_in_pg(&self) -> bool {
 		return (self.packet_idx + 1 + (self.bi.ends_with_continued as u8)) as usize
-			== self.bi.packet_positions.len();
+			== self.bi.packet_positions.len()
```

Comment:
Why do you remove the ; here? I like it.

Reply:
I just forgot to restore it.